Added pre-fitted model

CaMeLCa5e opened the pull request:
This is a more complete answer because it shows the fitted model as well as showing the new model.  I think that things are easier to follow like this.

CaMeLCa5e committed:
Added pre-fitted model

This is a more complete answer because it shows the fitted model as well as showing the new model.

amueller commented on doc/modules/ensemble.rst:
```diff
@@ -518,6 +517,14 @@ support ``warm_start=True`` which allows you to add more estimators to an alread
 fitted model.
 
 ::
+  >>>> from sklearn.metrics import mean_squared_error
```
this import seems unused?

amueller replied:
Or rather, it was imported before, right?

amueller commented:
Sorry I get it now, you copy and pasted code from above to make the example more self-contained. I think that makes sense as it is a new section in the docs. :+1: @pprett ?

CaMeLCa5e commented:
Yeah,
I don't think np was called again for the rest of the examples

And yes, I just took the pieces and made things more self contained.

I think the partial dependence section could be a little more self
contained as well. I will try to take a look at it.

Thanks,
Jack



On Thu, Apr 2, 2015 at 5:29 PM, Andreas Mueller <notifications@github.com>
wrote:

> Sorry I get it now, you copy and pasted code from above to make the
> example more self-contained. I think that makes sense as it is a new
> section in the docs. [image: :+1:] @pprett <https://github.com/pprett> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/4504#issuecomment-89049799>
> .
>

amueller commented:
travis is unhappy with your doctest.

CaMeLCa5e committed:
Took out import statement that was not used 

Took out unused import numpy statement.  We don't actually use np until line 237 (3.1.2.1. K-fold) and we do another import numpy statement on line 224.

amueller commented:
Travis is still unhappy

CaMeLCa5e committed:
Revert "Took out import statement that was not used "

This reverts commit 0b7b5f965a8d780cea31fb1f07f0522863ea0f5b.

CaMeLCa5e committed:
Revert "Added pre-fitted model"

This reverts commit 74ca5a2f9bdaf32f8046d28790a6234fba1c8dd8.
